Makefile

```diff
@@ -1,17 +1,31 @@
+PWD=$$(pwd)
+DOCKER_IMAGE="hashicorp/terraform-website:full"
+DOCKER_IMAGE_LOCAL="terraform-website-local"
+DOCKER_RUN_FLAGS=--interactive \
+		--rm \
+		--tty \
+		--volume "$(shell pwd)/website:/website/preview" \
+		--publish "3000:3000" \
+		-e "IS_CONTENT_PREVIEW=true" \
+		-e "NAV_DATA_PATH=./preview/data/cdktf-nav-data.json" \
+		-e "CONTENT_DIR=./preview/docs/cdktf"
+
 # Default: run this if working on the website locally to run in watch mode.
 website:
 	@echo "==> Downloading latest Docker image..."
-	@docker pull hashicorp/terraform-website:full
+	@docker pull ${DOCKER_IMAGE}
 	@echo "==> Starting website in Docker..."
-	@docker run \
-		--interactive \
-		--rm \
-		--tty \
-		--workdir "/website" \
-		--volume "$(shell pwd):/website/ext/terraform-cdk" \
-		--publish "3000:3000" \
-		hashicorp/terraform-website:full \
-		npm start
+	@docker run ${DOCKER_RUN_FLAGS} ${DOCKER_IMAGE} npm start
+
+website/local:
+	@echo "==> Starting website in Docker..."
+	@docker run ${DOCKER_RUN_FLAGS} ${DOCKER_IMAGE_LOCAL} npm start
+
+.PHONY: website/build-local
+website/build-local:
+	@echo "==> Building local Docker image"
+	@docker build https://github.com/hashicorp/terraform-website.git\#brk.feat/tfcdk-remote-content \
```

Comment:
TODO: update branch to `master` when https://github.com/hashicorp/terraform-website/pull/2209 is merged